data/json/npcs/refugee_center/surface_refugees/NPC_Alonso_Lautrec.json

```
@@ -409,6 +429,19 @@
       { "text": "Yeah, I'd better get going.", "topic": "TALK_DONE" }
     ]
   },
+  {
+    "type": "talk_topic",
+    "id": "TALK_REFUGEE_Alonso_Too_Young",
+    "dynamic_line": {
+      "npc_has_var": "Alonso_accent",
+      "type": "general",
+      "context": "none",
+      "value": "asked",
+      "no": "Alonso think maybe you're too young for Alonso.",
+      "yes": "Sorry kid, i can't do this, i still have some rules to follow."
```

```suggestion
      "yes": "Sorry kid, I can't do this, I still have some rules to follow."
```
This doesn't read to me as "Alonso doesn't want to commit statutory rape," but as "the Free Merchants/Old Guard will throw me in the woodchipper if I did."

---

You made a good point so I made it more clear Alonso has some ethical hiccups with the idea.